DB_setup.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE bootcampers (
```

Comment:
This is something you'll see all the time when you start working so glad you did this in your project!